packages/nextra/src/client/mdx-remote.tsx

```diff
@@ -8,19 +9,43 @@ export type MDXRemoteProps = Readonly<{
    * An object mapping names to React components.
    * The key used will be the name accessible to MDX.
    *
-   * @example `{ ComponentName: Component }` will be accessible in the MDX as `<ComponentName>`.
+   * @example
+   * `{ ComponentName: Component }` will be accessible in the MDX as `<ComponentName>`.
    */
   components?: MDXComponents
-  /** Pass-through variables for use in the MDX content. */
+  /**
+   * Pass-through variables for use in the MDX content.
+   * These variables will be available in the MDX scope.
+   */
   scope?: Scope
+  /**
+   * Raw JavaScript compiled MDX source code, a result of Nextra's
+   * [`compileMdx` function](https://nextra.site/api/compilemdx).
+   */
   compiledSource: string
 }>
 
-export function MDXRemote({
+/**
+ * A React component that renders compiled MDX content.
+ *
+ * @returns A rendered React element that renders the MDX content.
+ * @example
+ * ```mdx
```

Comment:
```suggestion
 * ```mdx filename="example.mdx"
```